### PR TITLE
Added Junit Tests for ResourceUtils

### PR DESCRIPTION
**Pull request by sonja-ct**
Signed-off-by: Sonja <sonja.matic@comtrade.com>

Added Junit Tests for ResourceUtils

**Related Issue**
/

**Description of the solution adopted**
/

**Screenshots**
/

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hey @sonja-ct,
can you please rebase to latest develop and push again? I think this is the reason for negative coverage, the CodeCov does not have the las coverage report. 
 EDIT: Sorry, my fault. I didn't have updated branch on my local machine so it looked like you did not rebase, but you did. I am looking into CodeCov to see if I can find anything specific.

File: commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java [hunk 1]
```diff
@@ -13,48 +13,112 @@
...
+        //Positive test
+        try {
+            assertEquals(url, ResourceUtils.getResource(resource));
+        } catch (AssertionError ex) {
+            fail("AssertionError not expected");
+        }
+
+        //Negative test
+        try {
+            assertEquals(url, ResourceUtils.getResource(noResource));
```
Comment (Coduz) [inline commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java]: If they are not intended to match please use assertNotEquals.
If `ResourceUtils.getResource(...)` is meant to throw an Exception catch the Exception and do asserts on the type of Exception expected.

File: commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java [hunk 2]
```diff
@@ -13,48 +13,112 @@
...
+    public void getResourceTest() {
+        String resource = "test.properties";
+        String noResource = "file:/does-not-exist";
+
+        ClassLoader classloaderTest = Thread.currentThread().getContextClassLoader();
+        URL url = classloaderTest.getResource(resource);
+
+        //Positive test
+        try {
+            assertEquals(url, ResourceUtils.getResource(resource));
```
Comment (Coduz) [inline commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java]: assertEquals has an overload method that accepts a message as a first argument. So fail invoked in the catch exception is not the correct usage of Assert stuff.
Please change this and also other similar usages

**Review by Coduz — CHANGES_REQUESTED** [review]
Hi @code-incubus ,

I've made two comments but those should be changed everywhere the same this is done.

Thanks,

_Alberto_

File: commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java (hunk 2 above)
Comment (sonja-ct) [reply commons/src/test/java/org/eclipse/kapua/commons/util/ResourceUtilsTest.java]: Hi @Coduz I have made all changes that you mentioned above. If something is not ok please let me know so I can fix it. Thanks

**Review by LeoNerdoG — APPROVED** [review]

**Comment by LeoNerdoG** [comment]
Hey @Coduz,
now this PR looks ok, I think this can be merged.

**Review by Coduz — APPROVED** [review]

**Pull request merged**